### DISCUSSION TITLE
Send Text - got wrong characters...

### DISCUSSION THREAD

**Discussion by AndreasWaldherr** [StreamController/StreamController#151] [General]
Hi! 

I try to send a Username (DOMAIN\Admnistrator) but when I paste it with my StreamDeck I got DOMAIN#Admnistrator.
I have an Linux Mint 21.3 installation in German Language with and German Keyboard Layout. 

In Ideas for me?

Thanks in advance
Andreas

**Comment by Core447** [comment]
Hey,

seems like you're having the same issue as described [here](https://github.com/StreamController/OSPlugin/issues/11). You can ready my comment on said post, for more information, but I'll fix this as soon as possible.

**Comment by AndreasWaldherr** [comment]
Thanks for your reply! Yes, this sounds like my problem...

Feel free to contact me in kind you need more details.



Core447 ***@***.***> schrieb am Mo., 1. Juli 2024, 20:29:

**Reply by Core447** [reply]
> Thanks for your reply!

No need to thank for that. I'm glad that you're using StreamController! 

> Feel free to contact me in kind you need more details.

Alright!

**Comment by AndreasWaldherr** [comment]
Do not hurry, have found a workaround!
To enter the \ correctly on my german keyboard I use the OS "Hotkey" function.

![Hotkey](https://github.com/StreamController/StreamController/assets/60932026/3e351ac8-004a-46d6-aa0c-5883e5399a5e)

Works great :-)

**Reply by Core447** [reply]
Yeah, it's working but displaying the wrong keys in the ui...
